src/backend/CassandraBackend.h

```diff
@@ -702,6 +723,12 @@ class CassandraBackend : public BackendInterface
     mutable std::uint32_t ledgerSequence_ = 0;
 
 public:
+    std::string
+    tablePrefix() const
+    {
+        return config_.valueOr<std::string>("table_prefix", "");
+    }
```

Comment:
i made this a public method back when i needed this for the migration. but i'm leaving it here because it seems totally reasonable to me for this to be a public method here.